HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # History
 ---
+## 1.5.0 / 2017-02-10
+- add `onDoubleClick` API
```

Comment:
No offense but that seems like a hack. It will work for sure, but having a double click as part of the API seems much cleaner and way more intuitive. Saying it's the recommended way seems like a bad argument because if that was the appropriate place for the double click then technically it's the appropriate place for the click too. We want 2 different ways to interact with the same component. You are simply nesting an element as a work around.    

I'm actually not using this module anymore but thought I'd add my 2 cents.